core/server/services/auth/api-key/index.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    admin: require('./admin'),
```

Comment:
I believe it is preferred that we export an object with getters. @kirrg001?

Reply:
👍 It's especially helpful if you have a lot of exports/requires in the target package.
This avoids requiring all the files on bootstrap. It's gets only required if you make use of the package notation on runtime e.g. `auth.apiKey.admin`